hw/pci-bridge/cxl_root_port.c

```diff
@@ -110,30 +125,79 @@ MemTxResult cxl_remote_cxl_mem_read(PCIDevice *d, hwaddr host_addr,
     if (cxl_packet == NULL) {
         release_packet_entry(tag);
         trace_cxl_root_debug_message("Failed to get CXL.mem MEM DATA response");
-        *data = 0xFFFFFFFF;
+        *data = 0xFF;
         return MEMTX_OK;
     }
 
-    *data = *(uint64_t *)(cxl_packet->data);
+    *data = *(uint8_t *)(cxl_packet->data);
     release_packet_entry(tag);
 
     return MEMTX_OK;
 }
 
+uint64_t cxl_get_dest_cache(PCIDevice *d, hwaddr host_addr, MemTxAttrs attrs)
+{
+    // TODO: maybe a lock mechanism here?
+    uint32_t cache_iter;
+    bool cache_hit = false;
+    uint64_t least_accessed_time = 0ULL - 1;
+    uint64_t least_accessed_candidate = 0;
+
+    for (cache_iter = 0; cache_iter < CXL_RW_NUM_BUFFERS; cache_iter++) {
```

Comment:
Let's call it `cache_idx` or `idx`.

`cache_iter` is more than a simple iterator looking at the below code.